### PR TITLE
Rename `comp` to `vdims` in field xarray methods

### PR DESCRIPTION
**Pull request by swapneelap**

### CODE REVIEW

File: discretisedfield/field.py [hunk 1]
```diff
@@ -3317,7 +3317,7 @@ def to_xarray(self, name="field", unit=None):
         """Field value as ``xarray.DataArray``.
 
         The function returns an ``xarray.DataArray`` with dimensions ``x``,
-        ``y``, ``z``, and ``comp`` (only if ``field.nvdim > 1``). The coordinates
+        ``y``, ``z``, and ``vdims`` (only if ``field.nvdim > 1``). The coordinates
```
Comment (lang-m) [inline discretisedfield/field.py:3320]: We should also generalise the spatial components.

File: discretisedfield/field.py [hunk 2]
```diff
@@ -3430,11 +3430,11 @@ def from_xarray(cls, xa):
         The class method accepts an ``xarray.DataArray`` as an argument to
         return a ``discretisedfield.Field`` object. The DataArray must have
         either three (``x``, ``y``, and ``z`` for a scalar field) or four
```
Comment (lang-m) [inline discretisedfield/field.py:3432]: This is also outdated as we can now have any number of spatial dimensions.

**Review by lang-m — APPROVED** [review]

**Comment by swapneelap** [comment]
@lang-m you are right about the docstrings. I was thinking of doing this in a separate PR dedicated to updating the docstrings. What do you think?

**Comment by lang-m** [comment]
You are right, we have a lot of outdated docstrings. So a separate PR sounds good.

**Pull request merged**